src/PostgREST/Config.hs

```diff
@@ -1,3 +1,4 @@
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
```

Comment:
Needed this for the `floatingOrInteger` thing later on. Would rather not suppress a warning though.